sites/all/themes/orwell/css/orwell.main.css

```diff
@@ -2817,6 +2830,44 @@ html.js input.form-autocomplete {
   content: "";
 }
 
+.ding-carousel .slick-dots {
+  background-color: #393939;
+  padding: 20px 0;
+  width: 100%;
+  text-align: center;
+  display: none !important;
+}
+
+.ding-carousel .slick-dots li {
+  display: inline-block;
+}
+
+.ding-carousel .slick-dots button {
+  background: none;
+  border: 0;
+  font-size: 0;
+}
+
+.ding-carousel .slick-dots button::before {
+  font-family: "icomoon";
+  speak: none;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  content: '';
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  content: "";
```

Comment:
Is this github or is it not escaped?

Reply:
Not escaped. That's how grunt does it in this project aparently.